addons/wowl/static/src/services/ui/ui.ts

```diff
@@ -0,0 +1,106 @@
+import { Component, core, tags, useState } from "@odoo/owl";
+import type { OdooEnv, Service } from "../../types";
+
+const { EventBus } = core;
+
+export interface UIService {
+  block: () => void;
+  unblock: () => void;
+}
+
+class BlockUI extends Component<{}, OdooEnv> {
+  static template = tags.xml`
+    <div>
+      <div t-if="state.blockUI" class="o_blockUI" >
+        <div class="o_spinner">
+            <img src="/wowl/static/img/spin.png" alt="Loading..."/>
+        </div>
+        <div class="o_message">
+            <t t-raw="state.message"/>
```

Comment:
Isn't that a security issue ?
The state.message may come from a translation, so potentially from a stranger.
I agree that I may be a bit paranoid here. :-)

Reply:
yep, it doesn't contain html, so t-esc does the trick here

Reply:
I think I'd remove the `<br />`'s and simply do t-esc

Reply:
But you need a line break if you want the same display. Is there a way to do it other than using html?